### PR TITLE
Camera Preset Not Resetting on Re-selection Bug

### LINKED ISSUE #9135: Camera Preset Not Resetting to Defined Position on Re-selection
**Describe the bug**

When selecting a preset in the camera feed on the patient consultation page, manually adjusting the camera position, and then clicking on the same preset again, the camera does not reset to the preset location as expected.

**To Reproduce**

1. Navigate to the Patient tab.
2. Search for the patient "Gigin" and click on their card.
3. Go to the Feed page and set a camera preset.
4. Manually move the camera to a different position.
5. Click on the same preset again.
6. Observe the issue.

**Expected behavior**

Clicking on the same preset should reset the camera feed to the preset's defined position.

**Screenshots**

Image

### PR DESCRIPTION
**Pull request by Jacobjeevan**
## Proposed Changes

- Fixes #9135 

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Enhanced management of camera presets, allowing independent selection and interaction with camera feeds.
	- Updated UI components to reflect the currently selected camera preset accurately.
	- Added new user interface messages for improved feedback, including notifications for linked assets and preset updates.

- **Bug Fixes**
	- Improved handling of camera movement, ensuring the selected preset is cleared appropriately when the camera is moved.

### CODE REVIEW

**Review by shivankacker — APPROVED** [review]

**Comment by nihal467** [comment]
image
image


- [ ] Now, when we try to modify the preset on the patient consultation page, its cut the feed in the page, as shown in the screenshot

**Comment by Jacobjeevan** [comment]
> image image
> 
>     * [ ]  Now, when we try to modify the preset on the patient consultation page, its cut the feed in the page, as shown in the screenshot

@nihal467 Try testing with presets present in the configuration page (in the drop down in the feed page); those should be working. You will notice that those presets (Bed 6, switches etc) don't show up on the configuration page (talking to Rithvik about what's happening here).

**Comment by Jacobjeevan** [comment]
> > image image
> > ```
> > * [ ]  Now, when we try to modify the preset on the patient consultation page, its cut the feed in the page, as shown in the screenshot
> > ```
> 
> @nihal467 Try testing with presets present in the configuration page (in the drop down in the feed page); those should be working. You will notice that those presets (Bed 6, switches etc) don't show up on the configuration page (talking to Rithvik about what's happening here).

Opened issue on BE for this: https://github.com/ohcnetwork/care/issues/2619

**Comment by nihal467** [comment]
LGTM

File: src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const ConsultationFeedTab = (props: ConsultationTabProps) => {
   const bed = props.consultationData.current_bed?.bed_object;
   const feedStateSessionKey = `encounterFeedState[${props.consultationId}]`;
   const [preset, setPreset] = useState<CameraPreset>();
+  const [selectedPreset, setSelectedPreset] = useState<CameraPreset>();
```
Comment (rithviknishad) [inline src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx:37]: this is now confusing... so what does `preset` state store?

File: src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx [hunk 2]
```diff
@@ -193,7 +194,7 @@ export const ConsultationFeedTab = (props: ConsultationTabProps) => {
                   feedStateSessionKey,
                   JSON.stringify({
                     type: "position",
-                    assetBed: preset.asset_bed.id,
+                    assetBed: "",
```
Comment (rithviknishad) [inline src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx]: if we are no longer going to cache asset bed id, we could remove that from the type and omit this right?

File: src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx (hunk 1 above)
Comment (Jacobjeevan) [reply src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx:37]: > this is now confusing... so what does `preset` state store?

selectedPreset specifically stores user's explicit selection, whereas preset is tied to the asset object.

https://github.com/ohcnetwork/care_fe/blob/66326dd053cffbb7020f4381ad397217b4de2698/src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx#L57

File: src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx (hunk 2 above)
Comment (Jacobjeevan) [reply src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx]: Hmm, yea we could remove that.

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**